OpenRA.Mods.Common/Traits/Buildings/Gate.cs

```diff
@@ -34,7 +34,7 @@ public class GateInfo : PausableConditionalTraitInfo, ITemporaryBlockerInfo, IBl
 	}
 
 	public class Gate : PausableConditionalTrait<GateInfo>, ITick, ITemporaryBlocker, IBlocksProjectiles,
-		INotifyAddedToWorld, INotifyRemovedFromWorld, INotifyBlockingMove, ISync
```

Comment:
There is a value with `[Sync]` (Position), why remove `ISync`? Tbh this doesn't look related so i guess this was a mistake.

Reply:
Makes sense.